### PR TITLE
Dpl 766 custom alert message 

### PR DESCRIPTION
**Pull request by seenanair**
Traction message refactoring to allow html message display using tailwind styling
 - enable_custom_alert feature flag to flip between bootstrap( blert) and html alert message
 - compatibility with all existing bootstrap features supported

### CODE REVIEW

File: src/components/TractionMessage.vue [hunk 1]
```diff
@@ -22,23 +49,47 @@ export default {
...
     },
+    /**
+     * Returns tailwind colors based on 'type' property with default being grey.
+     *  The supported values for 'type' property are as follows
+     * 'success' -  will be displayed in green color
+     * 'error'  -  will be displayed in red color
+     */
+    color() {
+      let value = { message: 'bg-gray-300 text-gray-400', icon: '' }
+      if (this.type === 'success') {
```
Comment (BenTopping) [inline src/components/TractionMessage.vue:82]: Might be nice to have all the types as an object or something where the keys are the type. Then the computed can just search the object and return the value or fallback onto the default.

**Review by BenTopping — APPROVED** [review]
Looks good, very seamless from the old alert. One small optional comment. Also Id be inclined to make the text slightly darker / more distinct from the background. But I have colour seeing issues anyway so probably worth a second opinion 😆

File: src/components/TractionMessage.vue (hunk 1 above)
Comment (seenanair) [reply src/components/TractionMessage.vue:82]: Sure, this seems like it will make it more confined. will try 👍

**Comment by seenanair** [comment]
> Looks good, very seamless from the old alert. One small optional comment. Also Id be inclined to make the text slightly darker / more distinct from the background. But I have colour seeing issues anyway so probably worth a second opinion 😆 

Thank you , Ben. I will try with a darker version, we are inclusive :)

**Review by stevieing — DISMISSED** [review]
Looks good to me. Checked it locally and all working.

**Comment by seenanair** [comment]
> Looks good, very seamless from the old alert. One small optional comment. Also Id be inclined to make the text slightly darker / more distinct from the background. But I have colour seeing issues anyway so probably worth a second opinion 😆
Done. Hope that's better now

**Review by stevieing — APPROVED** [review]
All working as expected :)

**Review by stevieing — APPROVED** [review]

**Pull request merged**